src/SwedbankPay/Path.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SwedbankPay;
+
+class Path
+{
+        /**
+    * This function is to replace PHP's extremely buggy realpath().
+    * https://stackoverflow.com/a/4050444/61818
+    *
+    * @param string The original path, can be relative etc.
+    * @return string The resolved path, it might not exist.
+    */
+    public static function resolve($path)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method resolve() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5303907)